rpm/sailjail-permissions.spec

```diff
@@ -10,6 +10,8 @@ BuildRequires: qt5-qmake
 BuildRequires: qt5-qttools-linguist
 BuildRequires: python3-base
 
+# TBD Conflicts: firejail < 0.9.63+git3
```

Comment:
This firejail dep still needs to be updated.

Btw, I'm assuming that the lack of Requires: firejail is intentional.

Reply:
After consultation: not needed as none of this has yet been in any release.